scripts/install-deps-rpm.sh

```diff
@@ -8,7 +8,6 @@ yum install \
   pkgconfig \
   glibc-devel \
   zeromq4-devel \
-  czmq-devel \
```

Comment:
Remove from Dockerfiles too so the next time we generate images this dependency is dropped?

Reply:
Or is czmq still required by some other dependency?

Reply:
Not that I know of but actually...maybe it makes sense to drop czmq in another PR just for release notes clarity?  This PR Is heavily about the cert class.